source/read/specify-a-query.txt

```diff
@@ -0,0 +1,274 @@
+criteria that MongoDB uses to match documents in a read or write operation.
+In a query filter, you can prompt the driver to search for documents that have
+an exact match to your query, or you can compose query filters to express more
+complex matching criteria.
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide run operations on the ``fruits`` collection,
+which contains documents representing fruits. The following
+code example shows how to create a database and collection, then
+insert the sample documents into your collection:
+
+.. literalinclude:: /includes/read/specify-queries.rb
+   :start-after: start-setup
+   :end-before: end-setup
+   :language: ruby
+   :dedent:
+   :copyable:
+
+Exact Match
+-----------
+
+Literal value queries return documents that have an exact match to your query filter.
+
+The following example specifies a query filter as a parameter to the ``find``
+method. The code returns all documents in which the value of the ``color`` field
+is ``'yellow'``:
+
+.. io-code-block::
+   :copyable: 
+
+   .. input:: /includes/read/specify-queries.rb
+      :start-after: start-find-exact
+      :end-before: end-find-exact
+      :language: ruby
+      :dedent:
+
+   .. output::
+      :visible: false    
+
+      {"_id"=>2, "name"=>"bananas", "qty"=>7, "rating"=>4, "color"=>"yellow", "type"=>["cavendish"]}
+      {"_id"=>4, "name"=>"pineapples", "qty"=>3, "rating"=>5, "color"=>"yellow"}
+
+.. note:: Find All Documents
+
+   To find all documents in a collection, call the ``find`` method
+   without passing any parameters:
+
+   .. code-block:: ruby
+
+      results = collection.find
+
+Comparison Operators
+--------------------
+
+Comparison operators evaluate a document field value against a specified value
+in your query filter. The following list describes common comparison operators:
+
+- ``$gt``: Returns documents in which the value of the given field is *greater than*
+  the specified value
+- ``$lte``: Returns documents in which the value of the given field is *less than or
+  equal to* the specified value
+- ``$ne``: Returns documents in which the value of the given field *does not equal* the
+  specified value
+
+.. tip::
+
+   To view a full list of comparison operators, see the :manual:`Comparison Query Operators
+   </reference/operator/query-comparison/>` guide in the {+mdb-server+} manual. 
+
+The following example specifies a comparison operator in a query filter as a
+parameter to the ``find`` method. The code returns all documents that have a
+``rating`` field value greater than ``2``:
+
+.. io-code-block::
+   :copyable: 
+
+   .. input:: /includes/read/specify-queries.rb
+      :start-after: start-find-comparison
+      :end-before: end-find-comparison
+      :language: ruby
+      :dedent:
+
+   .. output::
+      :visible: false
+
+      {"_id"=>1, "name"=>"apples", "qty"=>5, "rating"=>3, "color"=>"red", "type"=>["fuji", "honeycrisp"]}
```

Comment:
```suggestion
      {"_id": 1, "name": "apples", "qty": 5, "rating": 3, "color": "red", "type": ["fuji", "honeycrisp"]}
```
S: I've been using ``.to_json`` on results, so the output uses the usual colon syntax for key value pairs rather than the ruby arrow syntax. I can ask our group and engineering which makes most sense.

Reply:
I think asking in the group slack is a good idea, happy to change the output for consistency!